app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -33,7 +54,7 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
     { findContentWithElasticsearch, storeSearchFilter },
   ] = useContent();
   const navigate = useNavigate();
-  const [{ actions, frontPageImagesMediaTypeId }] = useLookup();
+  const [{ actions, frontPageImagesMediaTypeId, settings }] = useLookup();
```

Comment:
There is a `isReady` property to help with initialization issues and re-renders that should be used.   There are examples of this you can follow.